src/arch/src/aarch64/regs.rs

```diff
@@ -28,7 +28,7 @@ pub enum Error {
     /// Failed to Set multiprocessor state.
     SetMP(kvm_ioctls::Error),
     /// Failed to get a system register.
-    SetSysRegister(kvm_ioctls::Error),
+    SetRegister(kvm_ioctls::Error),
```

Comment:
On aarch64, the kernel divides the type of registers into core registers and system registers: https://elixir.bootlin.com/linux/v4.14.203/source/arch/arm64/kvm/guest.c#L295. If you take a look at the function saving the vcpu state on aarch64, it is split into 2 ([save_core_registers](https://github.com/firecracker-microvm/firecracker/blob/master/src/vmm/src/vstate/vcpu/aarch64.rs#L150) and [save_sys_registers](https://github.com/firecracker-microvm/firecracker/blob/master/src/vmm/src/vstate/vcpu/aarch64.rs#L152)) because the mechanism for calculating the reg ids is different. For the restore function however, we do not have this division, there is only one function: [restore_registers](https://github.com/firecracker-microvm/firecracker/blob/master/src/vmm/src/vstate/vcpu/aarch64.rs#L161) which restores both core and system registers so the name of the error was actually wrong before this change.